src/components/Button/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,224 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import Button, { ButtonProps, buttonSizes, buttonVariants } from '..'
+import { icons } from '../../Icon'
+
+export default {
+  component: Button,
+  title: 'Components/Button/Button',
+} as Meta
+
+const Template: Story<ButtonProps> = args => <Button {...args}>Button</Button>
+
+export const Default = Template.bind({})
+
+export const Variants = Template.bind({})
+Variants.parameters = {
+  docs: {
+    storyDescription: 'Set `variant` using variant property.',
+  },
+}
+Variants.decorators = [
+  () => (
+    <>
+      {buttonVariants.map(variant => (
+        <Button key={variant} variant={variant} mr={2} mb={2}>
+          {variant}
+        </Button>
+      ))}
+    </>
+  ),
+]
+
+export const Sizes = Template.bind({})
+Sizes.parameters = {
+  docs: {
+    storyDescription: 'Set `size` using size property.',
+  },
+}
+Sizes.decorators = [
+  () => (
+    <>
+      {buttonSizes.map(size => (
+        <Button key={size} size={size} mr={2}>
+          {size}
+        </Button>
+      ))}
+    </>
+  ),
+]
+
+export const Disabled = Template.bind({})
+Disabled.parameters = {
+  docs: {
+    storyDescription: 'Set `disabled` using disabled property.',
+  },
+}
+Disabled.decorators = [() => <Button disabled>Disabled</Button>]
+
+export const Progress = Template.bind({})
+Progress.parameters = {
+  docs: {
+    storyDescription:
+      'Add `ActivityIndicator` by setting progress property to `left` or `right`.',
+  },
+}
+Progress.decorators = [
+  () => (
+    <>
+      <Button progress="left" mr={2}>
+        left progress
+      </Button>
```

Comment:
```suggestion
export const Progress = Template.bind({})
Progress.parameters = {
args:{
progress: 'left'
},
  docs: {
    storyDescription:
      'Add `ActivityIndicator` by setting progress property to `left` or `right`.',
  },
}
```

Not enough ? 🤔

Reply:
Using `args` works well when you wanna show only one, once you wanna show multiple of them you need to use decorators